code/modules/projectiles/ammo_datums.dm

```diff
@@ -1020,13 +1020,13 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
 	hud_state = "laser_sniper"
 	damage = 42 //requires mod with -0.15 multiplier should math out to 40
 	max_range = 40
-	penetration = 20
+	penetration = 30
 
 /datum/ammo/energy/lasgun/M43/heat
 	name = "microwave heat bolt"
 	icon_state = "heavylaser"
 	hud_state = "laser_heat"
-	damage = 12 //requires mod with -0.15 multiplier should math out to 10
+	damage = 20 //requires mod with -0.15 multiplier should math out to 10
```

Comment:
Since this change removes attachements, remove any of the comments here that mention the old damage malus.

Reply:
this change doesn't actually remove attachments,it adds a new lasgun that has them as firemodes,but the M43 still exists and can still equip them attachies